crates/stages/src/stages/bodies.rs

```diff
@@ -88,8 +89,9 @@ impl<DB: Database, D: BodyDownloader> Stage<DB> for BodyStage<D> {
 
         // Cursors used to write bodies, ommers and transactions
         let mut body_cursor = tx.cursor_write::<tables::BlockBodies>()?;
-        let mut ommers_cursor = tx.cursor_write::<tables::BlockOmmers>()?;
         let mut tx_cursor = tx.cursor_write::<tables::Transactions>()?;
+        let mut ommers_cursor = tx.cursor_write::<tables::BlockOmmers>()?;
+        let mut withdrawals_cursor = tx.cursor_write::<tables::BlockWithdrawals>()?;
```

Comment:
Not sure where to put comment, but shudn't we check if withdrawals index is consistent across the blocks?
Left the comment in consensus, where index consistency can be check only in one block of withdrawals.
https://eips.ethereum.org/EIPS/eip-4895#system-level-operation-withdrawal

Reply:
@rkrasiuk is there any follow-up we need to do here?

Reply:
this is being worked on
https://github.com/paradigmxyz/reth/issues/1399

Reply:
ack thank you